docs/user-guide/layer-settings.md

```diff
@@ -1,120 +1,187 @@
 
-If you have any *editing grants* on the styles' source, you can customize layers' styles through the **Style Editor**.
+<img src="../img/layer-settings/ls-style.jpg" class="ms-docimage" style="max-width:450px"/>
 
-<img src="../img/layer-settings/style_editor.jpg" class="ms-docimage"/>
+In this case the user is allowed to:
 
-The Style Editor tool provides its functionalities on top of [GeoServer](http://geoserver.org/) REST APIs, so the styling
-functionalities are available only for those layers that are loaded in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) from GeoServer.
-By default the GeoServer's REST APIs are available only for the GeoServer administrators, so a basic authentication form will appears in MapStore to enter the *Admin* credentials. Without Admin rights, the editing capabilities on styles are not available and only a list of the available styles will be shown in order to assign one of them to the selected layer. If the [users integration between GeoServer and MapStore](../../developer-guide/integrations/users/geoserver/) is configured, the editing functionalities of the styles will be available according to the role of the authenticated user in MapStore in a more transparent way.
-<br>
-Once done these requirements' checks you can edit an existing style or create a new one from scratch.
+* Search through the available layer styles and select the desired one
 
-***Edit styles***
+* Create a new style
 
-* **Click** on the **Edit selected style** button <img src="../img/button/style_editor_edit_button.jpg" class="ms-docbutton"/> if you want to edit an existing style.
+* Edit an existing style
 
-<img src="../img/layer-settings/style_editor_edit_style.gif" class="ms-docimage"  style="max-width:700px;"/>
+* Delete an existing style
 
-* **Click** on the **Save current style** button <img src="../img/button/style_editor_save_current_style.jpg" class="ms-docbutton"/> to save your changes
+!!!note
+    The Style Editor tool, that allow the user to create a new style or edit/remove an existing one, provides functionalities on top of [GeoServer](http://geoserver.org/) REST APIs. This relationship implies that this tool:
 
-* **Click** on the **Back to style list** button <img src="../img/button/style_editor_back_button.jpg" class="ms-docbutton"/> to go back without saving the changes.
+    * Is available only for those layers that are loaded in [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) from [GeoServer](http://geoserver.org/)
 
-***Create new styles***
+    * Can be used only by Admin users because, by default, the GeoServer's REST APIs are available only for the GeoServer administrators. Take a look at the [User Integration with GeoServer](../../developer-guide/integrations/users/geoserver/) section of [Developer Guide](../../developer-guide) in order to understand how to configure the way MapStore and GeoServer share users, groups and roles
```

Comment:
By the default [service security rules](https://docs.geoserver.org/stable/en/user/security/service.html#service-security) the GeoServer's REST APIs are available only for the GeoServer administrators, so a basic authentication form will appears in MapStore to enter the *Admin* credentials. Without Admin rights, the editing capabilities on styles are not available and only the list of available styles will appear to allow the user to selct one of them to the layer.

Take a look at the [User Integration with GeoServer](../../developer-guide/integrations/users/geoserver/) section of [Developer Guide](../../developer-guide) in order to understand how to configure the way MapStore and GeoServer share users, groups and roles. If the users integration between GeoServer and MapStore is configured, the editing functionalities of the styles will be available according to the role of the authenticated user in MapStore in a more transparent way.